RELEASE/scripts/sl_ascend.ash

```diff
@@ -1598,7 +1598,10 @@ int handlePulls(int day)
 			{
 				pullXWhenHaveY($item[xiblaxian stealth cowl], 1, 0);
 			}
-			pullXWhenHaveY($item[Pantsgiving], 1, 0);
+			if (storage_amount($item[Pantsgiving]) > 0)
+			{
+				pullXWhenHaveY($item[Pantsgiving], 1, 0);
```

Comment:
Doesn't `pullXWhenHaveY` check if `historical_price` is greater than like... 20k or something? If Pantsgiving has somehow fallen to that price point, I think buying it isn't a bad idea :P

I could be remembering wrong though, and either way I'm fine with this change.

Reply:
You're totally right! I was confused because of a bug report with the line

```
Searching for "Pantsgiving"...
Search complete.
Do not have enough meat in Hagnk's to buy Pantsgiving. Need 240000000 have 230508002.
You must be a poor meatbag.
```

but yes, it wouldn't have actually purchased the item. Will improve the error logging in `pullXWhenHaveY`.